md5sum.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -ueo pipefail
+query_path=$(cat | jq -r '.path')
+md5=$(tar -cf - $query_path | md5sum| cut -d' ' -f1)
```

Comment:
This is not portable. E.g. on OSX it's just `md5` and not `md5sum`. Therefore, use `archive_file` resource instead.